package.json

```
@@ -0,0 +1,83 @@
+{
+  "name": "wp-starter-plugin",
```

To be sure, please perform a search against the entire plugin codebase so all instances of `wp-starter-plugin` are replaced with `clone-replace`.
```suggestion
  "name": "clone-replace",
```